README.md

```diff
@@ -6,20 +6,23 @@
 
 # nrich
 
-Nrich is a Java library developed at CROZ whose purpose is to make development of applications on JVM a little easier. It contains modules that were found useful on projects and as such were extracted
-to a common library. It is based on Spring Framework it also provides Spring Boot starters for most of the modules to make configuration easier.
+Nrich is a Java library developed at CROZ whose purpose is to make development of applications on the JVM a little
+easier.
+It was created by combining modules that were found useful on multiple projects into a common library.
+The library is based on the Spring Framework, and as such provides Spring Boot starters for most of the modules
+to make the configuration easier.
 
 Most modules are composed of multiple submodules:
 
 - api - has the `api` suffix and contains classes that represent API of the module (service interfaces, parameters, return types...)
 - implementation - contains actual api implementation
-- spring boot starter - has the `spring-boot-starter` suffix and contains Spring Boot auto configuration of the specified module
+- spring boot starter - has the `spring-boot-starter` suffix and contains Spring Boot autoconfiguration of the specified module
```

Comment:
Maybe we should use `auto-configuration` here. It is used in official Spring Boot documentation -> https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/

Reply:
I looked up the correct spelling and its sometimes written with a dash, and sometimes together. I agree thought that its better to use auto-cnofiguration as official Spring Boot documentation uses it too.